ClubDoorman/Worker.cs

```diff
@@ -626,6 +643,15 @@ await _bot.SendTextMessageAsync(
                             "Сообщение добавлено как пример спама в датасет",
                             replyToMessageId: replyToMessage.MessageId
                         );
+                        await badMessageManager.MarkAsBad(replyToMessage.Text ?? replyToMessage.Caption ?? string.Empty);
+                        break;
+                    case "/alwaysban":

```

Comment:
don't you think that every message that is banned deserves to be in that list? what I'm trying to say is I thought we don't need a separate command, but every time we handle Callback button with "ban" from admin chat this is enough human verification to forever tarnish such messages